logger/raven_logger.py

```diff
@@ -104,7 +104,10 @@ def capture(self, shell_args):
             extra['environment'] = dict([item.split('=', maxsplit=1) for item in shell_args.env.split('\n')])
 
         if shell_args.stderr:
-            extra['stderr'] = shell_args.stderr
+            # raven only accepts some number of list items
+            #  but you probably want the end of the message
+            stderr_list = shell_args.stderr.splitlines()[-self._client.list_max_length:]
```

Comment:
I installed a local copy of raven-bash and pulled up the python repl and check to find it.  I assumed they would have it.  I could make this much more defensive if you think it matters.